server/api/payment_handler.py

```diff
@@ -0,0 +1,142 @@
+from flask import Blueprint, jsonify, request, redirect, url_for
+from database import db
+from models import User, Payment
+from datetime import datetime
+from config import STRIPE_PUBLISHABLE_KEY_TEST, STRIPE_SECRET_KEY_TEST, STRIPE_CLIENT_ID_TEST
+from uuid import uuid4
+import stripe
+import secrets
+
+stripe.api_key = STRIPE_SECRET_KEY_TEST
+
+payment_handler = Blueprint('payment_handler', __name__)
+
+
+@payment_handler.route('/oauth')
+def get_oauth_link(user_id):
+    state_value = secrets.token_urlsafe()
+    # redirect_uri = 'http://localhost:3000/payments-demo'
+    redirect_uri = 'http://localhost:5000/users/1/payments/transfers/setup'
```

Comment:
you can save it in an env var so that when we launch this app, you can easily change the URI 

`REDIRECT_URI_BASE = 'http://localhost:5000"`